### PR TITLE
Create an acsf-tools-restore command to restore dumps.

### PR DESCRIPTION
**Pull request by vbouchet31**

### CODE REVIEW

File: acsf_tools.drush.inc [hunk 1]
```diff
@@ -213,6 +227,57 @@ function drush_acsf_tools_dump() {
...
+    $options = drush_get_context('cli');
+    unset($options['php']);
+    unset($options['php-options']);
+    unset($options['source-folder']);
+
+    foreach ($sites as $details) {
+      $domain = $details['domains'][0];
+      $prefix = explode('.', $domain)[0];
+
+      $source_file = $source_folder . '/' . $prefix . '.sql';
```
Comment (sdelbosc) [inline acsf_tools.drush.inc]: The files are likely to have been gzipped (from the dump command)

File: acsf_tools.drush.inc [hunk 2]
```diff
@@ -213,6 +227,57 @@ function drush_acsf_tools_dump() {
...
+
+      $command_drop = 'sql-drop';
+      $command_cli = 'sql-cli < ' . $source_file;
+
+      drush_log("\n=> Dropping and restoring database on $domain", 'ok');
+      drush_invoke_process('@self', $command_drop, $arguments, $options + ['l' => $domain, 'y']);
+      drush_invoke_process('@self', $command_cli, $arguments, $options + ['l' => $domain]);
+    }
+  }
+}
```
Comment (sdelbosc) [inline acsf_tools.drush.inc]: There may be a need of flushing memcache when at least one site got restored

File: acsf_tools.drush.inc (hunk 1 above)
Comment (vbouchet31) [reply acsf_tools.drush.inc]: Will add --gzip argument support.

Comment (vbouchet31) [reply acsf_tools.drush.inc]: @sdelbosc I added --gzip option.

File: acsf_tools.drush.inc (hunk 2 above)
Comment (vbouchet31) [reply acsf_tools.drush.inc]: @sdelbosc I am wondering if flushing memcache should not be the role of a different command. I would prefer to not block this command given there may be discussion around the need and best way to flush memcache.

Comment (sdelbosc) [reply acsf_tools.drush.inc]: fine for me

File: acsf_tools.drush.inc [hunk 3]
```diff
@@ -65,12 +65,14 @@ function acsf_tools_drush_command() {
...
       'source-folder' => 'The folder in which the dumps will be picked-up. Default to ~/drush-backups.',
+      'gzip' => 'Indicate the dump have been compressed using gzip. See acsf-tools-dump options.',
     ],
     'allow-additional-options' => TRUE,
     'bootstrap' => DRUSH_BOOTSTRAP_DRUPAL_SITE,
     'examples' => [
       'drush8 acsf-tools-restore' => 'Restore DB dumps for the sites of the factory. Default source folder will be used.',
-      'drush8 acsf-tools-restore --source_folder=/home/project/backup/20160617' => 'Restore DB dumps for the sites of the factory using files from the specified folder.'
+      'drush8 acsf-tools-restore --source_folder=/home/project/backup/20160617' => 'Restore DB dumps for the sites of the factory using files from the specified folder.',
+      'drush8 acsf-tools-restore --source_folder=/home/project/backup/20160617 --gzip' => 'Save as above but using compressed DB dumps.',
```
Comment (sdelbosc) [inline acsf_tools.drush.inc]: Same

**Pull request merged**